qa/tasks/cephadm.py

```diff
@@ -604,14 +604,13 @@ def ceph_osds(ctx, config):
             devs = devs_by_remote[remote]
             assert devs   ## FIXME ##
             dev = devs.pop()
-            short_dev = dev.replace('/dev/', '')
             log.info('Deploying %s on %s with %s...' % (
                 osd, remote.shortname, dev))
             _shell(ctx, cluster_name, remote, [
                 'ceph-volume', 'lvm', 'zap', dev])
             _shell(ctx, cluster_name, remote, [
                 'ceph', 'orch', 'daemon', 'add', 'osd',
-                remote.shortname + ':' + short_dev
+                remote.shortname + ':' + dev
```

Comment:
the changes to `qa/tasks/cephadm.py` need to go upstream first

if we take them downstream, they will become a source of rebase conflicts, which we do not want